index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+$db = new PDO('mysql:host=db; dbname=ukgamesfunded', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare("SELECT `title`, `year` , `theme` , `genre` , `rating`  FROM `game`");
+
+$query->execute();
+
+$result = $query->fetchAll();
+
+echo '<h2>UK Games Funded</h2>';
+
+?>
+<table>
```

Comment:
Don't forget about the rest of the HTML we need in order to have a valid webpage.

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
<-- Your table goes here -->
</body>
</html>
```

Every HTML page needs to have the above code, with everything visible (the table in your case) going inside the body tag.